src/ethereum/curve/contracts/interfacesV0/ICurveSwap.sol

```diff
@@ -39,6 +39,8 @@ interface ICurveSwap {
 
     function underlying_coins(uint256 arg0) external view returns (address);
 
+    function calc_withdraw_one_coin(uint256 _balance, int128 _tokenIndex) external view returns (uint256);
+
```

Comment:
Please update the ABI in `CurveSwap.json`, then you can merge after passing CI CD